restli-server/src/test/pegasus/com/linkedin/restli/server/defaults/RecordD.pdl

```diff
@@ -0,0 +1,8 @@
+namespace com.linkedin.restli.server.defaults
```

Comment:
`RecordD` is a duplicate of `RecordA`. Wouldn't it be simpler to have `RecordC` just contain a reference to `RecordA`?